### ISSUE TITLE
Rich text editor inserts font-family on <span> and <a> elements

### ISSUE THREAD

**Issue by adamshand** [pocketbase/pocketbase#3257]
When using the rich text editor, it periodically adds a font-family to `<a>` elements and wraps things in `<span>` elements which also have a font-family.  Eg.

```
<p><span style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;">Shortly after Andrea's paper, Ashleigh McCaull did a short piece about the </span><a style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;" href="http://localhost:5173/realistic-calendar">Realistic Calendar</a><span style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;">&nbsp;on Radio New Zealand's Counterpoint show,&nbsp;</span><a style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;" href="https://www.rnz.co.nz/national/programmes/checkpoint/audio/2018905181/wellington-s-spring-isn-t-as-bad-as-you-think-new-research">Wellington's spring isn't as bad as you think - new research</a>.</p>
```

I haven't been able to figure out the trigger for this. It only happens occasionally and on only on some content.

I think, but am not sure, that it's also sometimes inserting `&nbsp;` entities.

I'm currently only editing content with the PocketBase rich text form, so I think has to be the editor which is doing it.

**Comment by ganigeorgiev** [comment]
I suspect that the `&nbsp;` is when you enter new lines (see the example in https://www.tiny.cloud/docs/tinymce/6/wordcount/#character-count-logic).

The `span` and `style` attributes on the other hand I suspect are from copy-paste of the source text from some other html/document and the editor adds the extra styling in attempt to preserve the same formatting.

I'll consider with the next release to add an option to paste text as plain/nonformatted text.

**Comment by ganigeorgiev** [comment]
I've enabled the [`paste_as_text`](https://www.tiny.cloud/docs/tinymce/6/copy-and-paste/#paste_as_text) option to avoid messing with the page styles (_depending on the users demand in the future I may consider enabling the more detailed Edit toolbar but for now I don't want to clutter the UI unnecessary_).

The "fix" will be available with the next v0.18.0 release (_most likely sometime later this week_).

**Comment by ganigeorgiev** [comment]
On second thought disabling completely all pasted text formatting by default may not be desirable as it will prevent easily copy-paste of tables, links, lists, etc., so instead I've decided to attach to the TinyMCE `paste_postprocess` hook and filter the styles manually (_this actually is supported out of the box by TinyMCE in [PowerPaste](https://www.tiny.cloud/docs/tinymce/6/introduction-to-powerpaste/) but it is a paid plugin unfortunately_).

I've briefly tested it with content from LibreOffice Writer and the custom post processing seems work fine.

I also previously forgot to mention that if you want to paste a text without formatting you can also use `ctrl+shift+v`.
